freshmaker/pyxis.py

```diff
@@ -339,3 +339,32 @@ def image_is_tagged_auto_rebuild(self, nvr):
                 if set(tag['name'] for tag in repo['tags']) & set(auto_rebuild_tags):
                     return True
         return False
+
+    def remove_hotfix_images(self, bundle_nvrs):
+        """
+        Remove all hotfix images from a list of bundle_nvrs
+
+        :param list bundle_nvrs: list bundle_nvrs
+        :return: list of filtered bundle_nvrs
+        :rtype: list
+        """
+        for bundle_nvr in bundle_nvrs:
+            images = self.get_images_by_nvr(bundle_nvr, include=["data.parsed_data"])
+            if not images:
+                log.warning("Image %s is not found in Pyxis, ignore it.", bundle_nvr)
+                continue
+
+            hotfix_image_flag = False
+            labels = images["parsed_data"]["labels"]
```

Comment:
`images` are images for all available arches, you need to check the labels for one of the image (e.g. images[0]["parsed_data"]["labels"], or check if any of the images has the label, `images["parsed_data"]["labels"]` will raise error.
you can just do something like this:
```
# images for different arches should have same label names, so just check the first image
return any(label["name"] == "com.redhat.hotfix" for label in images[0]["parsed_data"]["labels"])
```

Reply:
I see, I added the [0] index in and used your way of any() to return a bool